partner_event/tests/test_event_registration.py

```diff
@@ -120,13 +121,10 @@ def test_data_update(self):
         self.partner_01.write({"email": "new@test.com"})
         self.assertEqual(event_2.registration_ids.email, "new@test.com")
 
+    @mute_logger("odoo.sql_db", "odoo.models.unlink")
```

Comment:
Why changing this if not benefit?

Reply:
Yeah, but the mute_logger was already added in the unlink operation. That's why I'm saying this.